src/views/about/License.vue

```diff
@@ -111,6 +111,12 @@ const licenseStatus = computed(() => {
 
 const { readFile } = useUploadFileAndRead()
 const handleUpload = async (file: any) => {
+  const nameStrings = file.name.split('.')
+  const type = nameStrings[nameStrings.length - 1]
+  if (type !== 'lic') {
+    EmqxMessage.warning(t('admin.uploadFileTypeError'))
+    return false
+  }
```

Comment:
How about use the `accept` field on upload component?

Reply:
I explain for her. if the `type` is set to `lic`, there will no restriction type directly, it seems there are no type values for `lic` file😥